db/migrate/20180531022247_add_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration[5.1]
+  def change
+    add_index :users, :email, unique: true
```

Comment:
does this mean add email as the index to the users table and ensure all emails are unique?

Reply:
Yup! It means that we can index the table by email and id (id is already included by default).